internal/pkg/library/library.go

```diff
@@ -70,12 +71,43 @@ func DownloadImageNoProgress(ctx context.Context, c *client.Client, imagePath, a
 	return DownloadImage(ctx, c, imagePath, arch, libraryRef, nil)
 }
 
+func getImageTags(image map[string]string) []string {
+	var ret []string
+	for t, _ := range image {
+		ret = append(ret, t)
+	}
+
+	return ret
+}
+
 // SearchLibrary searches the library and outputs results to stdout
 func SearchLibrary(ctx context.Context, c *client.Client, value string) error {
 	if len(value) < 3 {
 		return fmt.Errorf("bad query '%s'. You must search for at least 3 characters", value)
 	}
 
+	// If the user is searched for a container uri (eg. library/default/alpine) then
+	// try to get the image infomation, if unsuccessful, then search as usual.
+	if ref := strings.Split(value, "/"); len(ref) > 2 {
+		searchImage := value
+		if strings.HasPrefix(searchImage, "library://") {
+			searchImage = strings.TrimPrefix(searchImage, "library://")
+		}
+		sylog.Debugf("Attempting to get image info for: %s", searchImage)
+		cont, err := c.GetContainer(ctx, searchImage)
+		if err == nil {
+			fmt.Printf("Image:           %s\n", "library://"+searchImage)
+			fmt.Printf("Tags:            %s\n", getImageTags(cont.ImageTags))
+			fmt.Printf("Description:     %s\n", cont.Description)
+			// TODO: print if the image is signed or not
+			fmt.Printf("Total downloads: %d\n", cont.DownloadCount)
+			fmt.Printf("Stars:           %d\n", cont.Stars)
+			return nil
+		} else {
+			sylog.Verbosef("Failed to search container info: %s", err)
```

Comment:
I'm not sure we need to do this if we are going to fall back to the general search - will that always give output?